test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -1038,6 +1038,28 @@ def addSuccess(self, test):
     def _cleanup(self):
         self.logger.info('Cleaning up temporary files')
         du = DshUtils()
+        for user in PBS_USERS:
+            self.logger.info('Cleaning %s\'s home directory' % (str(user)))
+            runas = PbsUser.get_user(user)
+            for mom in self.param_dict['moms']:
+                ret = du.run_cmd(mom, cmd=['echo', '$HOME'], sudo=True,
+                                 runas=runas, logerr=False, as_script=True)
```

Comment:
Why are we cleaning the home directory only for MoM hosts? It will be effective only if MoM and Server are on the same machine. We should clean the home directory of the server host as well because the job submission is done from the host where pbs-server is running.

Reply:
Hi @sakshamgarg , You are right i will add server machine also to the list.

Reply:
@sakshamgarg addressed your comments. Please review.